src/Service.Tests/Unittests/ConfigValidationUnitTests.cs

```diff
@@ -1326,13 +1338,15 @@ private static void ValidateExceptionForDuplicateQueriesDueToEntityDefinitions(S
         /// <summary>
         /// Method to create a sample entity with GraphQL enabled,
         /// with given source and relationship Info.
+        /// Rest is disabled by default, unless specified otherwise.
```

Comment:
"Rest is disabled by default", only for this test and not in the context of the runtime config schema?

Reply:
Yes.